definitions/npm/express_v4.x.x/flow_>=v0.25.x/express_v4.x.x.js

```diff
@@ -0,0 +1,128 @@
+// @flow
+
+declare module 'express' {
+    declare type RouterOptions = {
+      caseSensitive?: boolean,
+      mergeParams?: boolean,
+      strict?: boolean
+    };
+    declare class RequestResponseBase {
+        app: Application;
+        get(field: string): string | void;
+    }
+    declare class Request extends http$IncomingMessage mixins RequestResponseBase {
+      baseUrl: string;
+      body: any;
```

Comment:
Should probably type this as `mixed` as it seems like a pretty significant source of types for most code